docs/source/demos/lead_scoring.ipynb

```diff
@@ -142,8 +142,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "from evalml.preprocessing import BalancedClassificationDataCVSplit\n",
+    "# we choose to slightly sample classes by setting the sampling ratio low\n",
+    "# we use the same splitter for both cases\n",
+    "data_splitter = BalancedClassificationDataCVSplit(sampling_ratio=0.1, random_seed=0)\n",
```

Comment:
Found that when we leave the data splitter as it was, the AUC for the LeadScoring-optimized search was 0.5 since it always guessed the same output. This is likely because balancing the data in this scenario leads to no learning done. If the data is balanced 1:4, as long as the gain outweighs the loss, we will always predict the same output. I changed the sampling to be smaller in order to highlight the point we wanted to convey. 

[This previous version]((https://evalml.alteryx.com/en/v0.18.0/demos/lead_scoring.html#Evaluate-on-hold-out)) is similar to our current performance

We note that we've had many iterations where the performance was very poor:
[v13](https://evalml.alteryx.com/en/v0.13.1/demos/lead_scoring.html#Describe-pipeline)
[v17](https://evalml.alteryx.com/en/v0.17.0/demos/lead_scoring.html#Evaluate-on-hold-out)
[v19](https://evalml.alteryx.com/en/v0.19.0/demos/lead_scoring.html#Evaluate-on-hold-out)
[v20](https://evalml.alteryx.com/en/v0.20.0/demos/lead_scoring.html#Evaluate-on-hold-out)

Reply:
I agree with @angela97lin and think this is worth a more directed research effort.  I think the last thing we want is someone to run the tutorial and not see what we're trying to show them, or worse, the opposite.  I submitted #2145 to cover this investigation as your issue was the short term solution to make the notebook satisfactory.

Reply:
Thanks for submitting the issue!